.github/workflows/pelican-version-test.yml

```diff
@@ -89,5 +89,4 @@ jobs:
           else
             # Check that GFM was NOT used
             ! grep '<p><del>Hi</del> Hello, <del>there</del> world!</p>' output/index.html
-          fi
-  
\ No newline at end of file
+          fi
```

Comment:
This didn't work completely, sorry. Please just fix it so there's a newline at the end of the file.